src/lava/lib/optimization/solvers/generic/solution_finder/process.py

```diff
@@ -15,23 +15,29 @@ def __init__(
         cost_diagonal,
         cost_coefficients,
         constraints,
+        backend,
```

Comment:
Also here: Is there any way to make the Process truly backend agnostic?

Reply:
no